tests/test_conduit_cli.py

```diff
@@ -0,0 +1,179 @@
+    mocked_ecs_client.register_container_instance(
+        cluster=mocked_cluster_arn, instanceIdentityDocument=mocked_instance_id_document
+    )
+    mocked_task_definition_arn = mocked_ecs_client.register_task_definition(
+        family="copilot-dbtunnel",
+        containerDefinitions=[
+            {"name": "test_container", "image": "test_image", "cpu": 100, "memory": 500, "essential": True}
+        ],
+    )["taskDefinition"]["taskDefinitionArn"]
+    mocked_ecs_client.run_task(cluster=mocked_cluster_arn, taskDefinition=mocked_task_definition_arn)
+
+    assert is_task_running(mocked_cluster_arn)
+
+
+@patch("os.system")
+@patch("commands.conduit_cli.is_task_running", return_value=True)
+def test_exec_into_task(is_task_running, system):
+    """Test that exec_into_task runs the `copilot task exec` command with
+    expected --app and --env flags."""
+
+    exec_into_task("dbt-app", "staging", "arn:random")
+
+    system.assert_called_once_with("copilot task exec --app dbt-app --env staging")
+
+
+@freeze_time("Jan 14th, 2020", auto_tick_seconds=60)
+def test_exec_into_task_timeout(capsys):
+    exec_into_task("dbt-app", "staging", "arn:random")
+
+    assert (
+        capsys.readouterr().out
+        == "Attempt to exec into running task timed out. Try again by running `copilot task exec --app dbt-app --env staging or check status of task in Amazon ECS console.\n"
+    )
+
+
+@mock_ecs
+@mock_resourcegroupstaggingapi
+@mock_sts
+def test_tunnel_no_cluster_resource(alias_session):
+    """Test that tunnel command prints an exit code when a cluster with app and
+    env tags is not found."""
+
+    runner = CliRunner()
+    result = runner.invoke(tunnel, ["--project-profile", "foo", "--app", "dbt-app", "--env", "staging"])
+
+    assert result.exit_code == 0
+    assert "No cluster resource found with tag filter values dbt-app and staging" in result.output
+
+
+@mock_sts
+def test_tunnel_profile_not_configured():
+    """Test that tunnel calls check_aws_conn and outputs expected error when AWS
+    profile variable isn't set."""
+
+    runner = CliRunner()
+    result = runner.invoke(tunnel, ["--project-profile", "foo", "--app", "dbt-app", "--env", "staging"])
+
+    assert 'AWS profile "foo" is not configured.' in result.output
+
+
+@mock_resourcegroupstaggingapi
+@mock_secretsmanager
+@mock_sts
+@patch("commands.conduit_cli.exec_into_task")
+@patch("commands.conduit_cli.create_task")
+def test_tunnel_task_not_running(create_task, exec_into_task, alias_session, mocked_cluster):
+    """Test that, when a task is not already running, command creates and execs
+    into a task."""
+
+    cluster_arn = mocked_cluster["cluster"]["clusterArn"]
+    secret_arn = boto3.client("secretsmanager").create_secret(
+        Name="/copilot/dbt-app/staging/secrets/POSTGRES", SecretString="secretivestring"
+    )["ARN"]
+    runner = CliRunner()
+    runner.invoke(tunnel, ["--project-profile", "foo", "--app", "dbt-app", "--env", "staging"])
```

Comment:
Just to separate the arranging and the action...
```suggestion
    
    runner.invoke(tunnel, ["--project-profile", "foo", "--app", "dbt-app", "--env", "staging"])
```